components/invitation_modal/invite_view.tsx

```diff
@@ -171,15 +186,7 @@ export default function InviteView(props: Props) {
                         defaultMessage={'Invite {inviteType} to {team_name}'}
                         values={{
                             inviteType: (
-                                props.inviteType === InviteType.MEMBER ?
-                                    <FormattedMessage
-                                        id='invite_modal.people'
-                                        defaultMessage='people'
-                                    /> :
-                                    <FormattedMessage
-                                        id='invite_modal.guests'
-                                        defaultMessage='guests'
-                                    />
```

Comment:
hey, so whats the reason for this change?

Reply:
@M-ZubairAhmed , sir actually the main reason behind these changes is that it produces a lint issue after running command "make fix style" and also formatmessage is the suggested way by all our senior devlopers . That's why I am using this here.